### DISCUSSION TITLE
build stuck forever

### DISCUSSION THREAD

**Discussion by azizjarrar** [vercel/next.js#25430] [Help]
my next app stay at building for ever 
![build](https://user-images.githubusercontent.com/46700725/119413535-b24f3480-bce5-11eb-8710-bda33e2a5dc4.PNG)
![Capture](https://user-images.githubusercontent.com/46700725/119413570-c4c96e00-bce5-11eb-86cc-480c0d9cf041.PNG)

**Comment by timneutkens** [comment]
Hey! Could you share a deployment url that can be investigated? Feel free to email support@vercel.com as well.

**Reply by azizjarrar** [reply]
hey timneutkens 
this is the url https://ahki-3ug18unxt-azizjarrar.vercel.app

**Reply by azizjarrar** [reply]
i past hours and hours rebuilding the web site line code by line in end i made it work in the end there are 2 problems sometime when you import component and you dont used the build in vercel don't work  (locally it work and even when i hosted in my vps it worked also ) and when i import DatePicker in some module it wont build like i imported in sing up it worked and when i imported in other page the build dont work i dont know why and how can you explain bc like i told you localy it work and in server ubuntu it works

**Reply by mcsdevv** [reply]
Hi @azizjarrarl, please contact Vercel Support [here](https://vercel.com/support/request) with a minimal reproduction of this issue and we'll take a look.
